scripts/lib/api/htmlToMd.ts

```diff
@@ -301,3 +319,76 @@ function buildMathExpression(node: any, type: "math" | "inlineMath"): any {
   }
   return { type: type, value };
 }
+
+function buildMdxComponent(h: H, node: any): any {
+  const apiType = node.tagName.charAt(0).toUpperCase() + node.tagName.slice(1);
+
+  const hastTree = {
+    type: "mdxJsxFlowElement",
+    name: apiType,
+    attributes: [
+      {
+        type: "mdxJsxAttribute",
+        name: "id",
+        value: node.properties.id,
+      },
+    ],
+    children: all(h, node),
+  };
+
+  addSimpleAttributeToHastTree(hastTree, "name", node.properties.name);
```

Comment:
Nit: consider calling this `maybeAddAttribute`

1. the `maybe` thing expresses this depends on if the attribute is defined
2. imo you can cut `toHastTree` because the first arg already makes that clear.